### ISSUE TITLE
dataset

### ISSUE THREAD

**Issue by fsh2017** [misogil0116/svpc#3]
may I get your processed dataset instead of the original dataset

**Comment by awkrail** [comment]
@fsh2017 You can accessed the dataset from https://github.com/misogil0116/svpc/tree/master/densevid_eval/yc2_data
It contains train/val/test set processed by us and also has ingredient annotations.

**Comment by fsh2017** [comment]
@misogil0116 thanks for your answer, But I still don't know where ''FEATURE_DIR'' and ''DURATION_DIR'' should point to during trainning, can you tell me exactly?

**Comment by awkrail** [comment]
@fsh2017 
First, please download All-in-one data of the YouCook2 dataset. Please click [All-in-one](http://youcook2.eecs.umich.edu/download) from this site.
Then, please unzip the dataset. After unzipping the YouCook2 dataset, you can find `youcook2/features` directory. It contains training, validation, and testing folders that store resnet and bn features. So, for FEATURE_DIR, please specify this directory.
```
[nishimura]nishimura/youcook2/features% ls
feat_csv  feat_dat  testing  training  validation  yc2
```

In `youcook2/features/yc2`, you can find `yc2_duration_frame.csv` file. For DURATION_DIR, please specify this file.
```
[nishimura] nishimura/youcook2/features/yc2% ls
val_yc2.json  yc2_annotations_trainval.json  yc2_duration_frame.csv
```
In summary, please specify as following in the above case:
```
FEATURE_DIR = nishimura/youcook2/features # your youcook2/features directory
DURATION_DIR = nishimura/youcook2/features/yc2_duration_frame.csv # your youcook2/features/yc2_duration_frame.csv
```

If you have further questions, please ask me.

**Comment by fsh2017** [comment]
@misogil0116 
Glad to see your reply! I'm having some problems.
I unzip the All-in-one dataset but there are only two folders in the features folder, i cant find testing、training、validation and yc2.
`fengs@LAPTOP-LAVT7JPL MINGW64 /d/workPlace/PythonWorkSpace/svpc/YouCookII/features (master)
$ ls
feat_csv/  feat_dat/
`

**Comment by awkrail** [comment]
@fsh2017 
OK, today, I will zip the feature/ directory and distribute it on my google drive.
After that I will notice you. Could you please wait for it?

**Comment by fsh2017** [comment]
@misogil0116 thank you!

**Comment by awkrail** [comment]
@fsh2017 Hi, I am sorry for my late reply. Now I uploaded [features.tar.gz](https://drive.google.com/file/d/1T5COAiqhIgqKvHzzsY2bw29fSuX68E39/view?usp=sharing) to Google drive and updated README.md. Could you try again?

**Comment by fsh2017** [comment]
@misogil0116 I have run your code successfully, thanks for your help!

**Comment by awkrail** [comment]
Enjoy!
